ReadME.md

```diff
@@ -0,0 +1,86 @@
+# Welcome to Rezume - a free Resume Generator
+
+[![first-timers-only](https://img.shields.io/badge/first--timers--only-friendly-tomato.svg?style=flat&logo=git)](https://github.com/CatsInTech/Rezume/issues) [![GitHub stars](https://img.shields.io/github/stars/CatsInTech/Rezume.svg?logo=github)](https://github.com/CatsInTech/Rezume/stargazers) [![GitHub forks](https://img.shields.io/github/forks/CatsInTech/Rezume.svg?logo=github&color=teal)](https://github.com/CatsInTech/Rezume/network) [![GitHub top language](https://img.shields.io/github/languages/top/CatsInTech/Rezume?color=yellow&logo=javascript)](https://github.com/vinitshahdeo/Water-Monitoring-System) 
+
+
+Rezume is a free online portal for generating resumes without worrying about the styling and formatting. In the web portal, you only need to input the basic information which you want to have on your resume and it will automatically generate a decent resume for you.
+
+## Major objective 
+
+- To let beginners venture into the domain of Web Development
+- To facilitate students and working professionals to create decent Resumes by focusing on the content rather than the styling and other UI features
+
+## Tech Stack 
+
+- **Frontend:** HTML5, CSS3, JavaScript, Bootstrap, ReactJS 
+- **Backend:** NodeJS
+
+## Installation Instructions
+
```

Comment:
Replace the above with something like this. Check your email for the complete instructions:

```
## Getting Started

### System Requirements

More in your email.
```

Reply:
I have updated the document, kindly check @praveenscience